demo/src/lib.rs

```diff
@@ -13,4 +13,26 @@
 ///     end
 /// ```
 /// The diagram is going to be located in place of the code snippet
+///
+/// Dark mode is automatically enabled if `dark` or `ayu` rustdoc theme is selected.
+/// You might need to reload the page after the theme change in order to redraw the diagram with an updated theme.
```

Comment:
> after the theme change

Either "after the theme changed" or "after the changing theme"

Reply:
Both of your options are grammatically incorrect but gotcha :)